Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/NMDataServiceTest.cs

```diff
@@ -514,5 +662,65 @@ private static Result<BatchAttributesSearchResponse> SetAttributeSearchNoYearWee
             };
             return attributeSearchResult;
         }
+
+        private static Result<BatchSearchResponse> SetSearchResultForWeeklyDuplicateFile()
```

Comment:
Just to be consistent with `SetSearchResultForDuplicateDailyFiles`
```suggestion
        private static Result<BatchSearchResponse> SetSearchResultForDuplicateWeeklyFiles()
```

Reply:
Done, Thanks